spec.html

```diff
@@ -4086,6 +4120,220 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
         </emu-alg>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-privatefield-record-specification-type">
+      <h1>The PrivateField Record Specification Type</h1>
+      <p>The PrivateField type is a Record used in the specification of private class fields.</p>
+      <p>Values of the PrivateField type are Record values whose fields are defined as by <emu-xref href="#table-privatefield-fields"></emu-xref>. Such values are referred to as <dfn>PrivateField Record</dfn>s.</p>
+      <emu-table id="table-privatefield-fields" caption="PrivateField Record Fields">
+        <table>
+          <tbody>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[PrivateName]]
```

Comment:
```suggestion
              [[PrivateFieldName]]
```
(and similarly throughout)
to parallel `[[PrivateFieldValue]]`